app/utils/accountHelpers.ts

```diff
@@ -163,16 +163,16 @@ export function getPublicAccountAmounts(
     if (!accountInfo) {
         return { total: 0n, staked: 0n, scheduled: 0n, atDisposal: 0n };
     }
-    const total = BigInt(accountInfo.accountAmount);
+    const total = BigInt(accountInfo.accountAmount.microCcdAmount);
     const staked =
         (accountInfo as AccountInfoBaker).accountBaker?.stakedAmount ??
         (accountInfo as AccountInfoDelegator).accountDelegation?.stakedAmount ??
         0n;
     const scheduled = accountInfo.accountReleaseSchedule
-        ? BigInt(accountInfo.accountReleaseSchedule.total)
+        ? BigInt(accountInfo.accountReleaseSchedule.total.microCcdAmount)
```

Comment:
This should already be sanitized by this point:
```suggestion
        ? accountInfo.accountReleaseSchedule.total.microCcdAmount
```